Adyen/Core/Payment Methods/AffirmPaymentMethod.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AffirmPaymentMethod.swift
+//  Adyen
+//
+//  Created by Naufal Aros on 7/6/21.
+//  Copyright © 2021 Adyen. All rights reserved.
+//
```

Comment:
hm.. do you have `swiftformat` ?
This is standard header

Reply:
I have installed it but it's not been triggered. Gonna take a look at it

Reply:
Fixed